src/server/enrollment-system.js

```diff
@@ -63,35 +63,97 @@ function veteranToPersonInfo(veteran) {
   };
 }
 
+function zeroPadNumber(number, padding) {
+  return (new Array(padding + 1).join('0') + number).slice(-padding);
+}
+
+function formDateToESDate(dateObject) {
+  if (dateObject.month >= 1 && dateObject.month <= 12 &&
+    dateObject.day >= 1 && dateObject.month <= 31 && // TODO: how robust does this need to be? Form validates as well.
+    dateObject.year > 1) {
+    return `${zeroPadNumber(dateObject.month, 2)}/${zeroPadNumber(dateObject.day, 2)}/${zeroPadNumber(dateObject.year, 4)}`;
+  }
+  return undefined;
+}
+
+function makeServiceBranch(serviceBranch) {
```

Comment:
`makeXXX` to me implies creation of an object or something.  In this case, we're doing a conversion.  Maybe `serviceBranchToSDSCode()` or something?

Same comment for the `makeDischargeType`.